data/ships.txt

```diff
@@ -2215,7 +2251,7 @@ ship "Osprey"
 	thumbnail "thumbnail/osprey"
 	attributes
 		category "Medium Warship"
-		"cost" 4400000
+		"cost" 3400000
```

Comment:
What's the reasoning behind this?
The Osprey is a bigger ship than the Corvette, able to take more of a punch and have more outfits. Why make it cheaper than a Corvette?

Reply:
I'd revert it, seeing as how the outfit change is only 192k extra on what it costs now, which doesn't really warrant a 1 mil decrease in hull cost. Maybe have 3 small jammers instead of 1 large, as that would actually end up reducing the cost of the current Osprey but still increase its jamming capabilities over its current loadout.